app/src/main/java/org/p2p/wallet/auth/interactor/UsernameInteractor.kt

```diff
@@ -57,7 +57,7 @@ class UsernameInteractor(
             }
             crashLogger.setCustomKey("username", usernameDetails?.username?.fullUsername.orEmpty())
         } catch (error: Throwable) {
-            Timber.i(error, "Failed to restore username for ${owner.base58Value}")
+            Timber.e(error, "Failed to restore username for ${owner.base58Value}")
```

Comment:
remove `owner.base58Value`

Reply:
why? we need to know which user is failed to restore username

Reply:
someone can easily pass an owner as a secret key to this function. 
we will know which user is failed from anywhere

Reply:
ok, removing